podpac/core/data/datasource.py

```diff
@@ -390,15 +382,19 @@ def _eval(self, coordinates, output=None, _selector=None):
                         step = 1
                     new_rsci.append(slice(mn, mx + 1, step))
                 else:
-                    new_rsci.append(slice(np.max(index), np.max(index) + 1))
+                    new_rsci.append(slice(index[0], index[0] + 1))
```

Comment:
Is this change because the index is now boolean.

Reply:
This is just for clarity. Maybe I missed something and it needs to be changed back (and explained). The `if` case is when the `index` has more than one element, and the `else` case is when it is has just one element. If it just has one element, what was the `np.max` for?

Reply:
Oh... that's because... uh... it's for `max` flavor. 

I blame the diff for now showing me the larger context. 

Thanks.